### DISCUSSION TITLE
Qodana licensing

### DISCUSSION THREAD

**Discussion by JuanCamiloRada** [JetBrains/Qodana#101] [Q&A] [answered]
What is the Qodana licensing?  can it be used without royalties in any type of project?

**Comment by brichbash** [comment, marked as answer]
Hello,

You may use our community and eap linters without any royalties.

You can view the license agreements for EAP linters at https://www.jetbrains.com/legal/docs/toolbox/user_eap/ and for Community linters at https://www.jetbrains.com/legal/docs/agreements/qodana/community-linters/.
